ui/components/multichain/app-header/app-header.scss

```diff
@@ -0,0 +1,52 @@
+.multichain-app-header {
+  flex-flow: column nowrap;
+  z-index: 55;
+  min-height: 64px;
+
+  &__contents {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    height: 64px;
+
+    @include screen-sm-max {
+      height: 100%;
+    }
+
+    @include screen-sm-min {
+      width: 85vw;
+    }
+
+    @include screen-md-min {
+      width: 80vw;
+    }
+
+    @include screen-lg-min {
+      width: 62vw;
+    }
```

Comment:
Note: We need a container component for the extension that uses these values so they aren't repeated throughout the app